### DISCUSSION TITLE
Online ImageMagick Software Error

### DISCUSSION THREAD

**Discussion by wlc77** [ImageMagick/ImageMagick#2874] [Help]
Getting a software error here: https://magickstudio.imagemagick.org/scripts/MagickStudio.cgi

**Comment by urban-warrior** [comment]
Try now.

**Reply by wlc77** [reply]
Thank you so much I appreciate it.

**Comment by wlc77** [comment]
Getting not found on server for a few days now. https://magickstudio.imagemagick.org/scripts/MagickStudio.cgi

**Comment by fmw42** [comment]
Works fine for me today!

**Reply by wlc77** [reply]
Yes, it's working again now.
